src/rabin.js

```diff
@@ -47,7 +47,13 @@ class Rabin {
         __release(lengthsPtr)
         __release(pointer)
```

Comment:
Much better swap this. So alloc and release will be in LIFO order.
```suggestion
        __release(pointer)
        __release(lengthsPtr)
```

Reply:
I changed to full, why __free what's the difference?

Reply:
All this described in docs: https://docs.assemblyscript.org/details/runtime#runtime-variants

in short `full` / `half` use RC/GC runtime, `stub`/`none` just manual memory managment and `retain / release` just stubs in this case

Reply:
Read that already I just can't find the __free function, I would prefer to use stub and manage memory manually.

Reply:
With full you still need to tell wasm to __retain and after __release so it's kinda manual. 
Any docs on how to go full manual with stub/none runtime?

Reply:
> With full you still need to tell wasm to __retain and after __release 

It's only for js / host side because we can't handle in this case memory automatically. But internally (in AS) you don't need call release / retain because compiler do this by himself (same as in Swift which used ARC)